<?xml version="1.0"?>
<launch>
  <arg name="debug"  default="False" />  
  <arg name="blur_param_"			default="1" />
  <arg name="hsv_target_"			default="145" />
  <arg name="hsv_threshold_"	default="10" />
  <arg name="contour_area_min_"	default="50" />
  <arg name="contour_area_max_"	default="500" />
  <arg name="contour_ratio_min_"	default="1" />
  <arg name="contour_ratio_max_"	default="15" />
  <arg name="aruco_detection_" 		default="False" />
  <arg name="image_topic_" 				default="/kinect2/sd/image_color_rect" />
  <arg name="cloud_topic_"				default="/kinect2/sd/points" />
  
  <node name="kinect_depth_test" pkg="ivan_aruco" type="kinect_depth_test" output="screen"> 
		<param name="blur_param_"          	type="int"   	value="$(arg blur_param_)"/> 
		<param name="hsv_target_"          	type="int"   	value="$(arg hsv_target_)"/> 
		<param name="hsv_threshold_"   			type="int"   	value="$(arg hsv_threshold_)"/> 
		<param name="contour_area_min_"    	type="int"   	value="$(arg contour_area_min_)"/> 
		<param name="contour_area_max_"     type="int"   	value="$(arg contour_area_max_)"/> 
		<param name="contour_ratio_min_"    type="double"   	value="$(arg contour_ratio_min_)"/> 
		<param name="contour_ratio_max_"    type="double"   	value="$(arg contour_ratio_max_)"/> 
		<param name="aruco_detection_"    	type="bool"   		value="$(arg aruco_detection_)"/> 
		<param name="image_topic_"					type="string"			value="$(arg image_topic_)"/>
		<param name="cloud_topic_"					type="string"			value="$(arg cloud_topic_)"/>
	</node>
	
</launch>


